### PR TITLE
Remove unused vars/procs in LC13 code

### PR DESCRIPTION
**Pull request by out-of-phaze**
## About The Pull Request
Removes a lot of unused vars and procs in LC13-specific code.

## Why It's Good For The Game
Like #948 but bigger. Specifically excludes the changes made in that PR. Shouldn't affect functionality because none of this is used. If you'd like me to change how I used or removed some vars please tell me.

### CODE REVIEW

File: ModularTegustation/tegu_items/lc13/refinery/sales.dm [hunk 1]
```diff
@@ -13,7 +13,6 @@
 	var/crates_per_box		//Just used to calculate examine text
 
 	var/generating
-	var/icon_full = "machinelcb_full"
```
Comment (Kitsunemitsu) [inline ModularTegustation/tegu_items/lc13/refinery/sales.dm:16]: If I ever design a different machine frame I will still need this.

File: code/game/objects/items/ego_weapons/non_abnormality/cane.dm [hunk 2]
```diff
@@ -12,7 +12,6 @@
 	var/charge_effect = "deal an extra attack in damage."
 	var/charge_cost = 2
 	var/charge
-	var/activated
```
Comment (Kitsunemitsu) [inline code/game/objects/items/ego_weapons/non_abnormality/cane.dm:15]: Leave this one in, These weapons were originally going to have a buff applied (and thus I would need to know if it was applied), and if I ever get around to it, they will in the future

File: code/modules/mob/living/simple_animal/abnormality/he/der_freischutz.dm [hunk 3]
```diff
@@ -71,6 +71,7 @@
 	datum_reference.qliphoth_change(3)
 	return ..()
 
+// currently unused, TODO: use or remove
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/he/der_freischutz.dm:74]: Admeme proc.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Couple things I likely am still using either right now, or in the very near future regarding refinery.

**Comment by EgorDinamit** [comment]
The conflicts!

**Comment by out-of-phaze** [comment]
> The conflicts!

There weren't any cornflakes a month ago when I requested a review :v

**Comment by Kitsunemitsu** [comment]
Stale, if you want to reopen, please do.

**Pull request closed without merging**